.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "MLPACK C++ Dev Container",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick a new MLPACK base image from
+		// https://hub.docker.com/u/mlpack
+		"args": { "VARIANT": "jenkins-amd64-debian" }
+	},
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			"settings": {},
+			"extensions": [
+				"twxs.cmake",
+				"ms-vscode.cpptools",
+				"ms-vscode.cmake-tools",
+				"llvm-vs-code-extensions.vscode-clangd"
+			]
+		}
+	}
+
+	// Switch to using root as a remote user
+	// details: https://aka.ms/dev-containers-non-root.
+	// "remoteUser": "root"
```

Comment:
What is the preference here?  I don't know about devcontainers, but should we be using the root user in the container?  (Easy enough to adapt the relevant dockerfiles if so.)

Reply:
For simplicity and to avoid folks wondering where user `mlpack` or `jenkins` was created, we probably want `root`.